### DISCUSSION TITLE
Help with TX16 internal modul

### DISCUSSION THREAD

**Discussion by robb1364** [EdgeTX/edgetx#1619] [General]
Hello, after flashing new 2,6 firmware my TX16S Eachine version show internal modul OFF and I can(t to make it on. When I flash it back to 2,5 everything works again. Any idea what can be wrong? Thank you

**Comment by pfeerick** [comment]
Most likely your internal RF setting under hardware has defaulted to OFF instead of MULTI...
https://user-images.githubusercontent.com/5500713/155244431-2ee9d5f3-99f5-49a1-a391-03aa8ffd88d1.mp4

**Reply by claudevandaele** [reply]
Had the same issue. For the Internal Module I was only able to activate in companion 2.6. This could not be activated in the hardware as you show in the video.

**Reply by pfeerick** [reply]
Can you elaborate further? Why not? MULTI not in the list? Couldn't get to the hardware tab?

**Reply by claudevandaele** [reply]
Hardware is available (TX16S) after update 2.6. Internal Multi was just disabled and not adjustable. I could only activate it in Companion. I also had to rebind with the receiver. The remarkable thing was that I could only adjust the external module without any problems after the 2.6 update in the hardware from the TX16S.

**Reply by pfeerick** [reply]
The ability to set the internal module is a new feature in 2.6, so if you were not running 2.6 on the TX16 that would explain why the option was missing. Otherwise, you should have been able to select MULTI from the hardware tab as shown, and then it would be available in the model settings. And depending on the protocol, you shouldn't have needed to rebind the receiver also - simply re-select the same receiver ID if it had model match.

**Reply by claudevandaele** [reply]
I had to search for a while and accidentally discovered the solution via companion. No other issues. Thanks for contributing ideas. Thx, Claude

**Comment by robb1364** [comment]
Hi Peter, everything is working now, so many thanks for your great work and support once again!
